SPDX-presentations.md

```diff
@@ -0,0 +1,6 @@
+This is a list of talks about SPDX with a link to info, slides, or video (as applicable). 
+
+Please add talks to the top of the list (so they are listed in reverse chronological order) in the following format:
+
+* "TITLE", NAME, EVENT, DATE, LINK TO SLIDES/VIDEO
```

Comment:
"Date" should be the first field, in YYY-MM-DD format, to facilitate automatic sorting

Change "Name" to "Presenter"

Reply:
Agreed during the Outreach Team meeting on 2022-10-17.